GoogleForCreators/web-stories-wp
# Dashboard: Updated Wellbeing template, improved blob imports

## Summary

This PR fixes the Wellbeing masking blobs.  Some were offset from center and the blob itself did not touch all corners of its bounding box.  This is because... I did a bad job importing them the first time 😱 

However, thanks to a nice Illustrator run down from Max, I was able to re-import the Wellbeing blobs perfectly centered and scaled to touch the bounding box 🎉 and this resulted in me having to update our Wellbeing template to adjust for the slight blob position changes that occurred.

## User-facing changes

- The `Wellbeing` template should appear as it did before.
- And 4 new blobs should appear in the Shapes panel of the Editor:
![image](https://user-images.githubusercontent.com/40646372/86643405-86be4100-bf91-11ea-90fb-d44a90d4f550.png)

## Testing Instructions

Wellbeing Template:
1.) Go to "Explore Templates".
2.) Click on "View" for the `Wellbeing` template.
3.) Make sure each page loads and looks goodly.

New Editor Blobs:
1.) Create a new story (or edit an existing one).
2.) Toggle to Shapes panel.
3.) Confirm that the 4 new blobs appear, and confirm that when added to the scene they're perfectly contained (no extra space) within their bounding box (like the images below).

<img src="https://user-images.githubusercontent.com/40646372/86644008-01875c00-bf92-11ea-9e65-675033b663be.png" height="100"><img src="https://user-images.githubusercontent.com/40646372/86644028-077d3d00-bf92-11ea-83ef-318b3453461a.png" height="100"><img src="https://user-images.githubusercontent.com/40646372/86644057-0e0bb480-bf92-11ea-916f-4ad358833dc9.png" height="100"><img src="https://user-images.githubusercontent.com/40646372/86644107-182db300-bf92-11ea-82d5-dd8d289f1ef0.png" height="100">
